.github/stale.yml

```
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: false
+
+# Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "Bug Fix"
+  - "Security"
+  - "Critical to release"
```

What about PR that are `block` by another PR and the latest is active?

---

Well we can't specify logic like "is the latest active?"

I don't feel much one way or the other about adding Blocked here. Blocked is more of a statement about the workflow of GitHub then it is about the contents of the code in the PR. The other labels listed here reflect types of code changes we absolutely don't want to miss.

---

Could we had `"blockec"`?

---

sure, I don't mind

---

Unresolving this because Blocked doesn't seem to be included.

---

Thanks for reviewing. I had included it at some point (pretty sure anyway). Maybe have been lost somehow in my rebasing.

```suggestion
  - "Critical to release"
  - "Blocked"
```

---

Can you also add "Blocking" so that things that are blocked don't get blocked forever if the blocker goes stale?